pkg/controllers/disruption/drift_test.go

```diff
@@ -553,8 +553,7 @@ var _ = Describe("Drift", func() {
 			Expect(ExpectNodes(ctx, env.Client)).To(HaveLen(1))
 			ExpectExists(ctx, env.Client, nodeClaim)
 		})
-		It("should drift nodes that have pods with the karpenter.sh/do-not-disrupt annotation when the NodePool's TerminationGracePeriod is not nil", func() {
-			nodeClaim.Spec.TerminationGracePeriod = &metav1.Duration{Duration: time.Second * 300}
+		It("should ignore nodes that have pods with the karpenter.sh/do-not-disrupt annotation when the NodePool's TerminationGracePeriod is not nil", func() {
```

Comment:
This changes the meaning of this test case -- did we mean to change this?

Reply:
Yes, this is capturing the case that a drift doesn't spin up a new node when there are blocking pods on the disrupted node. But let me make some changes to convey this a little better.